NoSoft-SA/nspack
# Feature/inspections - open-ended inspections refactored

Summary of this change:

Refactored inspections 


CHANGELOG entry:

### Added
- check_pallet(:failed_inspections)
- 
### Changed
- .
### Fixed
- .
### Removed
- services/failed_and_pending_pallet_inspections.rb
- check_pallet(:stock_grade)

How to test this code via the UI:

go path /finished_goods/inspection/inspections/inspection_results
select pallet ( still to be changed to accept scaner)
should list all applicable inspection_types for the pallet. 
add inspections 

PPECB is also handled with this improvement
